Dockerfile

```diff
@@ -12,11 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/distroless/static@sha256:aadea1b1f16af043a34491eec481d0132479382096ea34f608087b4bef3634be
-LABEL description="cert-manager operator for distributing trust bundles across a Kubernetes cluster"
+# Build the trust binary
+FROM docker.io/library/golang:1.17 as builder
```

Comment:
go.mod file still has Go 1.16, maybe you want to change that in this PR as well?

Reply:
Thanks, done!